### PR TITLE
Ardhika Softwares Added

### PR DESCRIPTION
**Pull request by gopithegeek**
Please review our [Contribution Guidelines](https://github.com/doomspork/elixir-companies/blob/master/CONTRIBUTING.md) to ensure your changes abides by it.

Changes should be made to the __end of the file__.

Review your changes and complete the checklist below.

- [x] Company name : Ardhika Software Technologies Pvt. Ltd.
- [x] Industry : Healthcare,Agriculture,Logistics,Travel,Hospitality & Technology Consulting
- [x] Company location : Chennai, India
- [x] Company description : We are a Product & Outsourcing company, specialises in open source technologies. Our preferred development platforms are GoLang, Elixir & VueJS. 
- [x] How Elixir is used : Elixir & Phoenix were used to develop a cloud based restaurant management system & Visitor Management System.

### CODE REVIEW

**Comment by gopithegeek** [comment]
Hi @doomspork Have added at the last as requested :) 👍

File: src/_data/companies.yml [hunk 1]
```diff
@@ -2059,3 +2059,15 @@ Bizneo HR:
   url: https://www.bizneo.com/
   location: Madrid, Spain
   description: "Bizneo’s HR Suite offers tools to manage your entire employee lifecycle. Built with Elixir and Phoenix, our HR management tools empower companies to act quickly and think globally in their HR processes."
+
+Ardhika Softwares:
+  industry: Computer Software/Engineering
```
Comment (doomspork) [inline src/_data/companies.yml]: One last thing @gopithegeek, it looks like the alignment for something here is off.  `industry` and `url` are not aligned.

**Comment by doomspork** [comment]
Thank you @gopithegeek!  I hope you have a wonderful day 😁

File: src/_data/companies.yml (hunk 1 above)
Comment (gopithegeek) [reply src/_data/companies.yml]: @doomspork thanks for bringing it to notice 👍 
have edited

**Pull request merged**